subsys/bluetooth/services/bas/bas_bcs.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024 Demant A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/services/bas.h>
+#include <zephyr/bluetooth/gatt.h>
+#include "bas_internal.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(bas, CONFIG_BT_BAS_LOG_LEVEL);
+
+#define BATTERY_CRITICAL_STATUS_CHAR_IDX 9
+
+/* Indicate battery critical status to all connections */
+static struct bt_gatt_indicate_params bcs_ind_params;
```

Comment:
Since this is only used in `bt_bas_bcs_update_battery_critical_status` we can move the variable in there